mne/io/nicolet/tests/test_nicolet.py

```diff
@@ -18,26 +16,5 @@
 
 def test_data():
     """Test reading raw nicolet files."""
-    tempdir = _TempDir()
-    raw = read_raw_nicolet(fname, preload=False)
-    raw_preload = read_raw_nicolet(fname, preload=True)
-    picks = [2, 3, 12, 13]
-    assert_array_equal(raw[picks, 20:30][0], raw_preload[picks, 20:30][0])
-
-    # Make sure concatenation works
-    raw2 = concatenate_raws([raw_preload.copy(), raw_preload])
-
-    # Test saving and reading
-    out_fname = op.join(tempdir, 'test_nicolet_raw.fif')
-    raw2.save(out_fname, tmax=raw.times[-1])
-    raw2 = Raw(out_fname)
-
-    full_data = raw_preload._data
-    data1, times1 = raw[:10:3, 10:12]
-    data2, times2 = raw2[:10:3, 10:12]
-    data3, times3 = raw2[[0, 3, 6, 9], 10:12]
-    assert_array_almost_equal(data1, full_data[:10:3, 10:12], 9)
-    assert_array_almost_equal(data1, data2, 9)
-    assert_array_almost_equal(data1, data3, 9)
-    assert_array_almost_equal(times1, times2)
-    assert_array_almost_equal(times1, times3)
+    raw = _test_raw_object(read_raw_nicolet, fname)
+    _test_raw_filter(raw, precision=5)
```

Comment:
usually `assert_allclose` is better than using `assert_array_almost_equal` because it's better at dealing with different magnitudes of data, so having `atol` and `rtol` arguments is better here